crates/host/src/wasmbus/mod.rs

```diff
@@ -334,12 +352,13 @@ struct Provider {
     image_ref: String,
 }
 
-/// Wasmbus lattice
+/// wasmCloud Host
 #[derive(Debug)]
-pub struct Lattice {
+pub struct Host {
     // TODO: Clean up actors after stop
     actors: RwLock<HashMap<String, Arc<Actor>>>,
     cluster_key: KeyPair,
+    config: HostConfig,
```

Comment:
The config should have been parsed on initialization, only the relevant parts from it should be stored in the resulting struct